azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/CallCompositeActivity.kt

```diff
@@ -228,6 +235,13 @@ internal class CallCompositeActivity : AppCompatActivity() {
     @SuppressLint("SourceLockedOrientationActivity", "RestrictedApi")
     private fun onNavigationStateChange(navigationState: NavigationStatus) {
         when (navigationState) {
+            NavigationStatus.NONE -> {
```

Comment:
This looks like a wrong place to make a decision about navigation status.

Reply:
Will go under refactor task.

Reply:
We decided not to remove this state. As we need to go through calllaunch action for both setup & skip setup screen feature because, localuser camera transmission state updates through that redux action.

Reply:
added in the refactor task